src/parser/ad_parser.js

```diff
@@ -329,20 +329,20 @@ function _parseExtension(extNode) {
   }
 
   // Only return not empty objects to not pollute extentions
-  return ext.isEmpty() ? null : ext;
+  return isEmpty(ext) ? null : ext;
 }
 
 /**
  * Parses the AdVerifications Element.
  * @param  {Array} verifications - The array of verifications to parse.
- * @return {Array<AdVerification>}
+ * @return {Array<createAdVerification>}
```

Comment:
it doesn't make much sense to assign this as a type, since it is a function. I would do
```
@return {Array<Object>} an array of ad verification objects
```